src/Support/Support.Autorest/test/Get-AzSupportCommunication.Tests.ps1

```diff
@@ -15,17 +15,44 @@ if(($null -eq $TestName) -or ($TestName -contains 'Get-AzSupportCommunication'))
 }
 
 Describe 'Get-AzSupportCommunication' {
-    It 'List' -skip {
-        { throw [System.NotImplementedException] } | Should -Not -Throw
+
+    It 'List' {
+        if($env.SupportPlanSubscription -eq "Basic support" || $env.SupportPlanSubscription -eq "Free"){
```

Comment:
There is a limitation due to support plan usage in 3rd party scenarios so even if we record separately, its a good check to have to avoid errors

Reply:
Could you elaborate more about this limitation?

Reply:
While recording tests separately we need to ensure that the test account being used is not a lower tier plan. This is because we have a limitation of allowing requests in 3rd party scenarios. If a valid test account is found then yes, this check can be removed.